requirements.txt

```diff
@@ -24,6 +24,7 @@ jupyter_client==7.3.4
 ldap3==2.9.1
 newrelic==8.8.0
 numpy>=1.21.0
+oracledb==1.3.1
```

Comment:
Could you add it to extra packages?
Also update setup.py https://github.com/mage-ai/mage-ai/blob/master/setup.py#L42

Reply:
Just added that on fb30b2cd00edfec210aec9086755bff2ca624359
Also did a `oracle` variable on setup.py